src/scanpy/tools/_cluster_resolution.py

```diff
@@ -0,0 +1,288 @@
+from __future__ import annotations
+
+import sys
+from typing import TYPE_CHECKING
+
+import pandas as pd
+
+if TYPE_CHECKING:
+    from collections.abc import Sequence
+    from typing import Literal
+
+    from anndata import AnnData
+
```

Comment:
Please mark the export. Also please rename the function to imperative mood (`find_` instead of `_finder`. A `…Finder` would be the name for a class!) 

```suggestion


__all__ = ["find_cluster_resolutions"]

```

Reply:
rename to 'find_cluster_resoltuon'